controllers/auth.go

```
@@ -86,18 +86,10 @@ func (c *ApiController) GetAccount() {
 }
 
 func (c *ApiController) GetUsers() {
-	var resp Response
 
 	//owner := c.Input().Get("owner")
 
-	users, err := auth.GetUsers()
-	if err != nil {
-		resp = Response{Status: "error", Msg: err.Error()}
-		c.Data["json"] = resp
-		c.ServeJSON()
-		return
-	}
-
+	users, _ := auth.GetUsers()
```

Ignoring error is not good.